### PR TITLE
Bug/MC-11914 Fix typo and layout issue

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
#### Changes made
Fix errors from my previous changes

### CODE REVIEW

**Review by AnKhangC — APPROVED** [review]

File: source/includes/kubernetes/_k8_persistentvolumes.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Retrieve a list of all persistent volumes in a given [environment](#administrati
 | `id` <br/>_string_ | The id of the persistent volume. This is the name of the resource. |
 | `metadata` <br/>_object_ | The metadata of the persistent volume.|
 | `spec` <br/>_object_ | The specification of the persistent volume. |
-| `spec.accessModes` <br/>_string_ | The volume can be mounted on a host in any way supported by the resource provider and will give the provider access to different capabilities. Values is one of `ReadWriteOnce` (by a single node), `ReadOnlyMany` (by many nodes) or `ReadWriteMany` (by many nodes). |
+| `spec.accessModes` <br/>_string_ | The volume can be mounted on a host in any way supported by the resource provider and will give the provider access to different capabilities. Value is one of `ReadWriteOnce` (by a single node), `ReadOnlyMany` (by many nodes) or `ReadWriteMany` (by many nodes). |
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_persistentvolumes.md:88]: Small suggestion for the `spec.persistentVolumeReclaimPolicy` attribute. Could we use the same format as `spec.accessModes`? Will make it easier to read.

Suggested
> One of `Retain` (manual reclamation), `Recycle` (basic scrub) or `Delete` (associated storage asset such as AWS EBS, GCE PD, Azure Disk, or OpenStack Cinder volume is deleted).

Current
> One of Retain - manual reclamation, Recycle - basic scrub or Delete - associated storage asset such as AWS EBS, GCE PD, Azure Disk, or OpenStack Cinder volume is deleted.

Comment (GuillaumeVialle) [reply source/includes/kubernetes/_k8_persistentvolumes.md:88]: It's already like that if I understand correctly what you're saying:
Screen Shot 2020-07-15 at 3 33 24 PM

Comment (patrickspensieri) [reply source/includes/kubernetes/_k8_persistentvolumes.md:88]: Yup my eyes fooled me

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/kubernetes/_k8_persistentvolumes.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/kubernetes/_k8_persistentvolumes.md:88]: 7144087dd70adcbe3ee2446b685b5b6532d76eb2

**Pull request merged**